src/test/java/com/google/cloud/tools/managedcloudsdk/install/ExtractorTest.java

```diff
@@ -21,46 +21,37 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.mockito.MockitoAnnotations;
-import org.mockito.invocation.InvocationOnMock;
-import org.mockito.stubbing.Answer;
+import org.mockito.junit.MockitoJUnitRunner;
 
+@RunWith(MockitoJUnitRunner.class)
 public class ExtractorTest {
 
   @Rule public TemporaryFolder tmp = new TemporaryFolder();
   @Mock private ProgressListener mockProgressListener;
   @Mock private ExtractorProvider mockExtractorProvider;
 
-  @Before
-  public void setupMocks() {
-    MockitoAnnotations.initMocks(this);
-  }
-
   @Test
   public void testExtract_success() throws Exception {
     final Path extractionDestination = tmp.newFolder("target").toPath();
     Path extractionSource = tmp.newFile("fake.archive").toPath();
 
     Mockito.doAnswer(
-            new Answer<Void>() {
-              @Override
-              public Void answer(InvocationOnMock invocationOnMock) throws Throwable {
-                Files.createDirectory(extractionDestination.resolve("some-dir"));
-                Files.createFile(extractionDestination.resolve("some-file"));
-                return null;
-              }
+            unused -> {
```

Comment:
ditto; don't introduce lambdas. Maybe it works in tests though if JavaNCSS doesn't have to instrument that?